pkg/service/repair/controller.go

```diff
@@ -74,7 +74,7 @@ func (c *rowLevelRepairController) shouldBlock(hosts []string, intensity int) bo
 
 	// DENY if any host has max nr of jobs already running
 	for _, h := range hosts {
-		if c.jobs[h] >= c.limits[h].Default {
+		if c.jobs[h] >= 1 {
```

Comment:
nit: the comment in line 75 is now inaccurate.

Reply:
thanks, updated the comment + removed another checks that are not relevant anymore